token-lending/sdk/src/math/decimal.rs

```diff
@@ -55,6 +55,11 @@ impl Decimal {
         Self(U192::from(percent as u64 * PERCENT_SCALER))
     }
 
+    /// Create scaled decimal from bps value
+    pub fn from_bps(bps: u64) -> Self {
```

Comment:
This (and the other `from_percent`) breaks at around 1800% since you multiply before calling `::from`. Just a heads up

Reply:
ah good catch. yeah i think we want to set the borrow weight as high as possible. will fix